docs/Options.md

```diff
@@ -170,7 +170,6 @@ Git status indicators is shown only when you have dirty repository.
 | `SPACESHIP_GIT_STATUS_UNTRACKED` | `?` | Indicator for untracked changes |
 | `SPACESHIP_GIT_STATUS_ADDED` | `+` | Indicator for added changes |
 | `SPACESHIP_GIT_STATUS_MODIFIED` | `!` | Indicator for unstaged files |
-| `SPACESHIP_GIT_STATUS_RENAMED` | `»` | Indicator for renamed files |
```

Comment:
Actually, I like the rename indicator and I think it's pretty useful. I wouldn't agree to remove it.

Reply:
I like it too, but this PR changes the behavior of git_status icons to only show icons for unstaged changes, and unfortunately git doesn't have a concept of "unstaged renamed" files - it shows them as "deleted and created" 😞

Reply:
As a user, I remember having the rename indicator even with the old git status 'mechanism' (that is, the one proposed back by @maximbaz). Why would it need to be removed now?

Reply:
![image](https://user-images.githubusercontent.com/1177900/35562668-5b877fc2-05b4-11e8-88ed-f97dd8be8683.png)